src/test/java/edu/ksu/canvas/attendance/services/SynchronizationServiceUTest.java

```diff
@@ -301,54 +305,6 @@ public void synchronizeStudentsFromCanvasToDb_UpdateExistingStudentInCourse() th
         assertEquals(expectedDeleted, expectedStudentInDb.getDeleted());
     }
 
-    @Test
-    public void synchronizeStudentsFromCanvasToDb_UpdateDeletedStudentInCourse_StayDeleted() throws Exception {
-        Long previousCanvasSectionId = 350L;
-        Long previousCanvasCourseId = 350L;
-        String previousSisUserId = "uniqueSisId";
-        String previousName = "Doe, Jane";
-        AttendanceStudent expectedStudentInDb = new AttendanceStudent();
-        expectedStudentInDb.setCanvasCourseId(previousCanvasCourseId);
-        expectedStudentInDb.setCanvasSectionId(previousCanvasSectionId);
-        expectedStudentInDb.setSisUserId(previousSisUserId);
-        expectedStudentInDb.setName(previousName);
-        expectedStudentInDb.setDeleted(Boolean.TRUE);
-        List<AttendanceStudent> studentsInDbForCourse = new ArrayList<>();
-        studentsInDbForCourse.add(expectedStudentInDb);
-
-        Long expectedCanvasSectionId = 250L;
-        Long expectedCanvasCourseId = 550L;
-        String expectedSisUserId = "uniqueSisId";
-        String expectedName = "Smith, John";
-        Integer expectedStudentsSavedToDb = 1;
-        Boolean expectedDeleted = Boolean.TRUE;
-
-        User user = new User();
-        user.setSisUserId(expectedSisUserId);
-        user.setSortableName(expectedName);
-        Enrollment enrollment = new Enrollment();
-        enrollment.setUser(user);
-        List<Enrollment> enrollments = new ArrayList<>();
-        enrollments.add(enrollment);
-        Section section = new Section();
-        section.setId(expectedCanvasSectionId);
-        section.setCourseId(expectedCanvasCourseId.intValue());
-        Map<Section, List<Enrollment>> canvasSectionMap = new HashMap<>();
-        canvasSectionMap.put(section, enrollments);
-
-        when(mockStudentRepository.findByCanvasCourseId(expectedCanvasCourseId)).thenReturn(studentsInDbForCourse);
-        when(mockStudentRepository.save(any(AttendanceStudent.class))).thenReturn(expectedStudentInDb);
-        List<AttendanceStudent> actualStudents = WhiteboxImpl.invokeMethod(synchronizationService, "synchronizeStudentsFromCanvasToDb", canvasSectionMap, anyBoolean());
-
-        verify(mockStudentRepository, atLeastOnce()).save(expectedStudentInDb);
-        assertThat(actualStudents.size(), is(equalTo(expectedStudentsSavedToDb)));
-        assertSame(expectedStudentInDb, actualStudents.get(0));
-        assertEquals(expectedCanvasSectionId, expectedStudentInDb.getCanvasSectionId());
-        assertEquals(expectedCanvasCourseId, expectedStudentInDb.getCanvasCourseId());
-        assertEquals(expectedSisUserId, expectedStudentInDb.getSisUserId());
-        assertEquals(expectedName, expectedStudentInDb.getName());
-        assertEquals(expectedDeleted, expectedStudentInDb.getDeleted());
-    }
 
     @Test
     public void synchronizeStudentsFromCanvasToDb_DroppedStudent() throws Exception {
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Define and throw a dedicated exception instead of using a generic one. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonar.ome.ksu.edu/coding_rules#rule_key=squid%3AS00112)
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Rename this method name to match the regular expression '^[a-z][a-zA-Z0-9]*$'. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonar.ome.ksu.edu/coding_rules#rule_key=squid%3AS00100)